server/config.go

```diff
@@ -13,16 +14,24 @@ type Threshold struct {
 }
 
 type Graph struct {
-	Name            string      `json:"name"`
-	Title           string      `json:"title"`
-	Description     string      `json:"description"`
-	GraphType       string      `json:"graphType"`
-	MetricName      string      `json:"metricName"`
-	ColorSchemes    []string    `json:"colorSchemes"`
-	Thresholds      []Threshold `json:"thresholds"`
-	QueryExpression string      `json:"queryExpression"`
-	YAxisUnit       string      `json:"yAxisUnit"`
-	ValueRounding   int         `json:"valueRounding"`
+	Name                 string       `json:"name"`
+	Title                string       `json:"title"`
+	Description          string       `json:"description"`
+	GraphType            string       `json:"graphType"`
+	MetricName           string       `json:"metricName"`
+	ColorSchemes         []string     `json:"colorSchemes"`
+	Thresholds           []Threshold  `json:"thresholds"`
```

Comment:
Graphs already have `Thresholds` field. I think that the `request` and `limit` metrics are a better fit for `Thresholds` rather than `SubMetrics` as suggested. However the current `Threshold` struct just allow a static value currently. Please investigate how the `Threshold` field gets rendered in the UI and confirm if we can enhance this field to accommodate our use-case. If so, my suggestion is to move the `QueryExpression` and `Label` from the `SubMetrics` struct to the `Threshold`.

Reply:
Built a completely different logic to handle the Req and Limits